main.go

```diff
@@ -1,18 +1,18 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"log"
 
-	"github.com/birdsean/review-droid/github"
+	"github.com/birdsean/review-droid/comments"
+	"github.com/birdsean/review-droid/github_client"
```

Comment:
Suggestion: consider grouping the imports into standard library, 3rd party, and project-specific packages